sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/impl/ConditionalPreconditionsTest.java

```diff
@@ -651,6 +657,535 @@ void testDescriptionModificationModificationNoDescriptors() {
         reset(mockManipulations);
     }
 
+    /**
+     * Tests whether DescriptionModificationMdsDescriptorPrecondition correctly calls manipulations
+     * in case a RemovableDescriptor is present.
+     */
+    @Test
+    @DisplayName("DescriptionModificationMdsDescriptorPrecondition correctly calls manipulation when a "
+            + "RemovableDescriptor is present.")
+    public void testDescriptionModificationMdsDescriptorPreconditionGoodPresentDescriptors() {
+
+        // given
+        final var descriptor1Handle = "superHandle1";
+        final var descriptor2Handle = "handle;Süper;2";
```

Comment:
descriptor2Handle is not really used in this unit test.

Reply:
Removed.